SB2018/2018-02-14.md

```diff
@@ -0,0 +1,36 @@
+---
+to: All Teams
+subject: Tech day reminder - Saturday 17th Feb
+---
+
+Hi all,
+
+We've got another tech day this Saturday (17th Feb)!
+
+Tech days are a great opportunity for you to get robotics advice and help with technical issues from our volunteers. We may also have a tour of some university facilities as well.
+
+## Location
+
+We're currently trying to confirm a better venue for the tech day, but in the mean time, here are the details for our venue if we don't confirm the new one.
+
+Unless we confirm a new location, the tech day will be held in [Building 25](http://data.southampton.ac.uk/building/25.html) at the University of Southampton's Highfield Campus:
```

Comment:
It might be useful to specify explicitly that this is the same location as the last one, so people may be able to remember locations etc?